torchtune/models/clip/_component_builders.py

```diff
@@ -104,8 +104,7 @@ def clip_vision_encoder(
         VisionRotaryPositionalEmbeddings(
             patch_size=patch_size,
             tile_size=tile_size,
-            max_num_tiles=max_num_tiles,
-            dim=head_dim // 2,
+            dim=head_dim,
```

Comment:
doesn't this affect Llama 3.2 as well? or does that not use max num tiles? is the `head_dim // 2` pulled up to the caller function?

Reply:
See [here](https://github.com/pytorch/torchtune/pull/2570/files#r2038401036)